charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 3.69.4
+
+* Simplify OTel Agent default pipelines.
```

Comment:
The PR description mention that it might break previous users (still ok because the feature is still in beta). But could you reflect this information in the changelog entry, because by reading this change log it looks very safe to apply this change.

Reply:
I'd argue that these changes might break previous users 🙂 :

1. All the customers currently participating in the private beta program have their own OTel configs, none of them are relaying on the default config.
2. The simplified pipeline is 100% backward compatible with previous version: the data coming to DD ingest with the new simplified config is identical to the data from previous config.

Nonetheless, I've bumped the minor version as you suggested.